spec/presenters/hyrax/collection_presenter_spec.rb

```diff
@@ -225,6 +227,8 @@
     subject { presenter.total_viewable_collections }
     let(:ability) { double(::Ability, user_groups: ['public'], current_user: user) }
     let(:user) { create(:user) }
+    let(:collection) { FactoryBot.create(:collection_lw) }
```

Comment:
Same here RE: `collection_lw`.  Should it be...

```suggestion
    let(:collection) { FactoryBot.create(:hyrax_collection) }
```

Reply:
i don't think these two suggestions will pass. as noted in the commit messages, these run afoul of explicit `ActiveFedora::Relation#accessible_by` calls in the presenter implementation, and those need to be addressed first.

that's a bigger project.

Reply:
Ah.  I missed that.  How would you like to handle this?  I'm fine merging this now and fixing the other problem afterward.  Is there an issue to reference?